app/src/main/java/com/peacecorps/pcsa/MainActivity.java

```diff
@@ -33,13 +37,24 @@
     private List<String> listDataHeader;
     private HashMap<String, List<String>> listDataChild;
     private static final String TAG = MainActivity.class.getSimpleName();
+    public static String FRAGMENT_TAG = MainActivityFragment.TAG;
+    private FragmentManager fragmentManager = getSupportFragmentManager();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        Fragment mainActivityFragment = new MainActivityFragment();
-        swapFragmentIn(this,mainActivityFragment,MainActivityFragment.TAG);
+        if(savedInstanceState != null)
```

Comment:
I had to write these lines to handle orientation changes. 
When the orientation changes, the activity is recreated. So, here I am just using the fragment from the backstack instead of recreating it